### PR TITLE
added renting

### PR DESCRIPTION
**Pull request by MicLi01**
- added in image field in Items model
- added basic renting functionality (will add on to this later)

### CODE REVIEW

File: server/controllers/items.js [hunk 1]
```diff
@@ -28,8 +29,18 @@ const getAll = (req, res, next) => {
       .catch(next);
 };
 
+const rent = async (req, res, next) => {
```
Comment (oscarchong) [inline server/controllers/items.js]: If you're using VS code, you can add on an "eslint" extension that can help you fix all the eslint errors.

File: server/routes/api/items.js [hunk 2]
```diff
@@ -27,4 +27,13 @@ router.get('/:itemId', [
   validatorErrors,
 ], itemController.get);
 
+/**
+ * @memberof module:api/items
+ * @name POST /rent
+ */
+router.get('/rent/:itemId', [
+  param('itemId').isInt(),
```
Comment (synzen) [inline server/routes/api/items.js]: slight fix, should be `isAlphanumeric()` (#52 )

File: server/controllers/items.js (hunk 1 above)
Comment (MicLi01) [reply server/controllers/items.js]: good to know

File: server/routes/api/items.js (hunk 2 above)
Comment (MicLi01) [reply server/routes/api/items.js]: nice catch. correction has been made.

**Pull request merged**